packages/mongodb-rag-core/src/contentStore/EmbeddedContent.ts

```diff
@@ -101,6 +101,15 @@ export type QueryFilters = {
   sourceType?: Page["sourceType"] | string[];
 };
 
+/** 
+  Metadata of data source
+*/
+export type DataSourceMetadata = {
+  id: string;
+  version?: string[];
```

Comment:
```suggestion
  version?: {label: string, isCurrent: boolean }[];
```

pls track the current version in here as well. something like the above typing